### PR TITLE
[Av1] added ffmpeg-qsv and gst-msdk av1 8bit lp encode cqp/cbr/vbr run2run support

### PR DESCRIPTION
**Pull request by FocusLuo**
1, ffmpeg-qsv run2run
2, gst-msdk run2run

### CODE REVIEW

File: test/ffmpeg-qsv/encode/av1.py [hunk 1]
```diff
@@ -53,6 +54,13 @@ def test(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cqp_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
```
Comment (uartie) [inline test/ffmpeg-qsv/encode/av1.py:59]: `spec` -> `spec_r2r`

File: test/ffmpeg-qsv/encode/av1.py [hunk 2]
```diff
@@ -75,6 +83,12 @@ def test(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, pr
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```
Comment (uartie) [inline test/ffmpeg-qsv/encode/av1.py:88]: `spec` -> `spec_r2r`

File: test/ffmpeg-qsv/encode/av1.py [hunk 3]
```diff
@@ -98,3 +112,9 @@ def init(self, tspec, case, gop, bframes, tilecols, tilerows, bitrate, fps, qual
   def test(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
+
+  @slash.parametrize(*gen_av1_vbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```
Comment (uartie) [inline test/ffmpeg-qsv/encode/av1.py:118]: `spec` -> `spec_r2r`

File: test/gst-msdk/encode/av1.py [hunk 4]
```diff
@@ -57,6 +58,13 @@ def test(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cqp_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
```
Comment (uartie) [inline test/gst-msdk/encode/av1.py:63]: `spec` -> `spec_r2r`

File: test/gst-msdk/encode/av1.py [hunk 5]
```diff
@@ -79,6 +87,12 @@ def test(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, pr
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```
Comment (uartie) [inline test/gst-msdk/encode/av1.py:92]: `spec` -> `spec_r2r`

File: test/gst-msdk/encode/av1.py [hunk 6]
```diff
@@ -102,3 +116,9 @@ def init(self, tspec, case, gop, bframes, tilecols, tilerows, bitrate, fps, qual
   def test(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
+
+  @slash.parametrize(*gen_av1_vbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```
Comment (uartie) [inline test/gst-msdk/encode/av1.py:122]: `spec` -> `spec_r2r`

**Review by uartie — CHANGES_REQUESTED** [review]
wrong spec used for r2r init

File: test/gst-msdk/encode/av1.py (hunk 6 above)
Comment (FocusLuo) [reply test/gst-msdk/encode/av1.py]: thanks review and catch this issue, updated

File: test/gst-msdk/encode/av1.py (hunk 5 above)
Comment (FocusLuo) [reply test/gst-msdk/encode/av1.py]: thanks review and catch this issue, updated

File: test/ffmpeg-qsv/encode/av1.py (hunk 1 above)
Comment (FocusLuo) [reply test/ffmpeg-qsv/encode/av1.py]: thanks review and catch this issue, updated

File: test/ffmpeg-qsv/encode/av1.py (hunk 2 above)
Comment (FocusLuo) [reply test/ffmpeg-qsv/encode/av1.py]: thanks review and catch this issue, updated

File: test/ffmpeg-qsv/encode/av1.py (hunk 3 above)
Comment (FocusLuo) [reply test/ffmpeg-qsv/encode/av1.py]: thanks review and catch this issue, updated

File: test/gst-msdk/encode/av1.py (hunk 4 above)
Comment (FocusLuo) [reply test/gst-msdk/encode/av1.py]: thanks review and catch this issue, updated

**Review by uartie — APPROVED** [review]
LGTM

**Pull request merged**